jicoco/src/main/java/org/jitsi/rest/AbstractJettyBundleActivator.java

```diff
@@ -406,20 +406,35 @@ private Connector initializeConnector(Server server)
                 JETTY_SSLCONTEXTFACTORY_NEEDCLIENTAUTH,
                 false);
 
-            sslContextFactory.setExcludeCipherSuites(
-                "SSL_RSA_EXPORT_WITH_DES40_CBC_SHA",
-                "SSL_DHE_RSA_EXPORT_WITH_DES40_CBC_SHA",
-                "SSL_DHE_DSS_EXPORT_WITH_DES40_CBC_SHA",
-                ".*NULL.*",
-                ".*RC4.*",
-                ".*MD5.*",
-                ".*DES.*",
-                ".*DSS.*");
+            /* Mozilla Guideline v5.4, Jetty 9.4.15, intermediate configuration
+               https://ssl-config.mozilla.org/#server=jetty&version=9.4.15&config=intermediate&guideline=5.4
+               */
+            /* TLS 1.3 requires Java 11 or later. */
+            String version = System.getProperty("java.version");
+            if (version.startsWith("1.")) {
+                version = version.substring(2, 3);
+            } else {
+                int dot = version.indexOf(".");
+                if (dot != -1) { version = version.substring(0, dot); }
+            }
+            int javaVersion = Integer.parseInt(version);
+
+            if (javaVersion >= 11) {
+                sslContextFactory.setIncludeProtocols("TLSv1.2", "TLSv1.3");
+            }
+            else {
+                sslContextFactory.setIncludeProtocols("TLSv1.2");
+            }
             sslContextFactory.setIncludeCipherSuites(
```

Comment:
Does this excluded ciphers not in the list? Does that mean the existing `setExcludeCipherSuites` call was not needed?

Reply:
You can either include or exclude -- the old `setExcludeCipherSuites` call was needed because there wasn't an explicit include list.  (If you have an include list with patterns you may also want an exclude list, but this include list is an explicit list isntead.)

Reply:
While you're touching this anyway, wouldn't it be a good time to introduce a property for the ciphers? Use the ones now listed as default.

Reply:
What's the use case for this? I feel like it'd be better for us to keep the compiled-in list of ciphers correct (with PRs if we should be changing them), rather than foist the responsibility onto deployments.

Reply:
To let whoever wants it enable/disable additional ciphers, e.g. to force EC only. Ideally though this list came from the OS and not from within the (any) application. I can't say how important it is to make it a property, but having a TLS cipher list hardcoded just _feels_ wrong.

Reply:
I agree making this configurable would be good, but I don't think we should block this PR on it